app/middlewares/authentication.js

```diff
@@ -21,7 +21,11 @@ exports.verifyToken = (req, res, next) => {
     try {
       req.userPayload = authService.decodeToken(token);
       return userService.findOne({ id: req.userPayload.id }).then(user => {
-        if (user && req.userPayload.generatedDate > user.baseAllowedDateToken) {
+        if (
+          user &&
+          req.userPayload.generatedDate > user.baseAllowedDateToken &&
+          req.userPayload.exp > Date.now()
```

Comment:
exp is already checked by [jwt-simple](https://github.com/hokaccha/node-jwt-simple/blob/master/lib/jwt.js#L101), use that